k8s.tf

```diff
@@ -64,3 +64,34 @@ resource "kubernetes_stateful_set" "elasticsearch" {
     }
   }
 }
+
+
+resource "kubernetes_cron_job" "backup_cleanup" {
```

Comment:
Could this be by any chance scheduled by `slm` as well? And also, can we make backups optional (if so, with optional predefined cronline)?

Reply:
No, I think SLM can do only snapshots and their retention. We can make backup optional, tho I don't we don't ever want to turn them off :)